site/_data/docs/extensions/toc.yml

```diff
@@ -1,6 +1,13 @@
 - url: /docs/extensions/mv3
 - url: /docs/extensions/whatsnew
-- url: /docs/extensions/gethelp
+- url:
+- title: i18n.docs.extensions.support
+  sections:
+  - url: /docs/extensions/support-feedback/
```

Comment:
Assume you might be working on this but just a note that we seem to have added a few things by accident :)

Reply:
My branch is a little older than the current state of the repo. I don't know why this part of the navigation appears. It's not a difference with main.